frontend/src/metabase/visualizations/components/ChartSettings/ChartSettings.unit.spec.tsx

```diff
@@ -134,42 +128,4 @@ describe("ChartSettings", () => {
 
     expect(screen.queryByText("Foo")).not.toBeInTheDocument();
   });
-
-  it("reset settings should revert to the original card settings with click behavior", async () => {
```

Comment:
Why can we remove this whole test?

Reply:
ah that's supposed to be temporary. It should be moved to the `DashboardChartSettings` because I've moved the visualization there.